app/app.go

```diff
@@ -351,9 +350,6 @@ func New(
 		packetforwardkeeper.DefaultForwardTransferPacketTimeoutTimestamp, // forward timeout
 	)
 
-	// Token filter wraps packet forward middleware and is thus the first module in the transfer stack.
-	transferStack = tokenfilter.NewIBCMiddleware(transferStack)
```

Comment:
[blocking]
1. Please remove x/tokenfilter module if it is no longer needed
2. Please update specs to remove this line: https://github.com/celestiaorg/celestia-app/blob/6928c85dc740a858bf072e31cbb2048e7bbae88b/specs/src/state_machine_modules_v5.md?plain=1#L11